### PR TITLE
Fixing some typos & asking questions

### PR DESCRIPTION
**Pull request by jesusgonzalez12**
Aside from fixing a few typos, I changed "StreamApp" reference to "Stream Worker". If you consider it should be "stream worker" is better, go ahead.

A few other questions I have is:
1) I did not understand the line "Here, the keys of payload mapping can be defined using the dot notation as a.b.c, or using any constant string value as '$abc'.", and I cannot see an example of that use case. If this goes here, an example would greatly help. 2) I think the line  "unavailable or coursing errors" should be "unavailable or causing errors".
3) We have this in one query: "CREATE SOURCE <source_name> WITH (type = 'source_type'". Shouldn't it be "... WITH (source.type = ..."?

### CODE REVIEW

**Comment by diana-macrometa** [comment]
@StoyanTc, can you answer the questions in the description?

File: docs/cep/query-guide/stream.md [hunk 1]
```diff
@@ -68,8 +68,8 @@ The following is the list of source types supported by Stream:
 
 |Source type | Description|
 | ------------- |-------------|
-| `database` | Allow stream app to consume events from collections (doc, graphs) running in the same or different geo fabric. |
-| `stream` | Allow stream app to consume events from streams (local, geo-replicated) running in the same or different geo fabric. |
+| `database` | Allow the Stream Worker to consume events from collections (doc, graphs) running in the same or different geo fabric. |
```
Comment (diana-macrometa) [inline docs/cep/query-guide/stream.md]: Do not capitalize stream worker unless you are referring to the UI, such as "click **Stream Worker**.

File: docs/cep/query-guide/stream.md [hunk 2]
```diff
@@ -227,7 +227,7 @@ Distributed sink provides a way to publish Stream events to multiple endpoints i
 
 **Syntax**
```
Comment (diana-macrometa) [inline docs/cep/query-guide/stream.md]: Change these bold lines pretending to be headings into actual headings.

File: docs/cep/query-guide/stream.md [hunk 3]
```diff
@@ -419,7 +419,7 @@ Stream infers and automatically defines the fault stream of `TempStream` as give
 CREATE STREAM !TempStream (deviceID long, roomNo int, temp double, _error object);
 ```
 
-The StreamApp extending the above the use-case by adding failure generation and error handling with the use of [queries](#query) is as follows.
+The Stream Worker extends the above use-case by adding failure generation and error handling with the use of [queries](#query) is as follows.
 
 Note: Details on writing processing logics via [queries](#query) will be explained in later sections.
```
Comment (diana-macrometa) [inline docs/cep/query-guide/stream.md]:
```suggestion
:::note
Details on writing processing logics via [queries](#query) will be explained in later sections.
:::
```

File: docs/cep/query-guide/stream.md [hunk 4]
```diff
@@ -419,7 +419,7 @@ Stream infers and automatically defines the fault stream of `TempStream` as give
 CREATE STREAM !TempStream (deviceID long, roomNo int, temp double, _error object);
 ```
 
-The StreamApp extending the above the use-case by adding failure generation and error handling with the use of [queries](#query) is as follows.
+The Stream Worker extends the above use-case by adding failure generation and error handling with the use of [queries](#query) is as follows.
```
Comment (diana-macrometa) [inline docs/cep/query-guide/stream.md]:
```suggestion
The stream worker extends the above use case by adding failure generation and error handling with the use of [queries](#query) as follows.
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]

**Comment by StoyanTc** [comment]
> @diana-macrometa @jesusgonzalez12  - 
1. I cannot understand your question, please provide a link to which part of doc you're pointing to?
2. The original statement is correct, ref. https://docs.google.com/document/d/1Qbgkhl44AYOmkSp6JfT_iDL9IVYRxRFE/edit

File: docs/cep/query-guide/stream.md (hunk 1 above)
Comment (jesusgonzalez12) [reply docs/cep/query-guide/stream.md]: ok, thanks.

File: docs/cep/query-guide/stream.md (hunk 2 above)
Comment (jesusgonzalez12) [reply docs/cep/query-guide/stream.md]: changing for "Syntax". Is that right?

**Comment by jesusgonzalez12** [comment]
Hi @StoyanTc 

1) Here https://macrometa.com/docs/cep/query-guide/stream#sink on "Map Payload" subsection, last paragraph.
2) OK, good to know, thanks.

**Comment by diana-macrometa** [comment]
What is the status of this, @jesusgonzalez12 ?

**Comment by jesusgonzalez12** [comment]
Thanks for keeping an eye @diana-macrometa.

The only thing pending is my question about the added paragraph I've been talking about, my point number one question in a previous comment.

@StoyanTc, I have provided the location of the paragraph. Would be great to have more context or an example.

**Comment by StoyanTc** [comment]
> Thanks for keeping an eye @diana-macrometa.
> 
> The only thing pending is my question about the added paragraph I've been talking about, my point number one question in a previous comment.
> 
> @StoyanTc, I have provided the location of the paragraph. Would be great to have more context or an example.

@jesusgonzalez12 - 

- If the key is `$abc='some value'` the output will be `{"$abc": "some value"}`. This is just a constant.
- If the key is `a.b.c="some value", the output will be nested json like `{"a" : {"b":{"c":"some value"}}}`

**Comment by jesusgonzalez12** [comment]
Thanks @StoyanTc.

@diana-macrometa, adding Stoyan's example can improve de doc. What do you think?

**Review by diana-macrometa — APPROVED** [review]
Looks good!

**Pull request merged**